File: spring/prog_terminal_marittimo/src/main/java/progetto_terminal_marittimo/richiesta.java
package progetto_terminal_marittimo;

public class richiesta {
    private int ID;
    private int id_cliente;
    private int id_merce;
    private int quantita;


    public richiesta(int id, int id_c, int id_m, int q){
        this.ID=id;
        this.id_cliente=id_c;
        this.id_merce=id_m;
        this.quantita=q;
    }

    public int getID(){
        return this.ID;
    }

    public int getId_cliente(){
        return this.id_cliente;
    }

    public int getId_merce(){
        return this.id_merce;
    }

    public int getQuantita(){
        return this.quantita;
    }
}

File: pagine/js/richiesta.js
class Richiesta {
  constructor(id, id_cliente, id_merce, quantita) {
    this.ID = id;
    this.id_cliente = id_cliente;
    this.id_merce = id_merce;
    this.quantita = quantita;
  }

  getID() {
    return this.ID;
  }

  getId_cliente() {
    return this.id_cliente;
  }

  getId_merce() {
    return this.id_merce;
  }

  getQuantita() {
    return this.quantita;
  }
}
